### ISSUE TITLE
Cell abstraction for merging members

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#207]
### Problem

Some members might want to be treated as a unit.

### Solution

Anywhere the UI would normally naively operate directly on members, instead operate on a "cell" abstraction (typically a cell of 1 member), per yesterday's convo with @vijayee.

**Comment by snowteamer** [comment]
Members who might want to be treated as a unit regarding their income details and payments, might still want to act as individuals in the context of chat DMs or non-monetary contributions

**Comment by taoeffect** [comment]
Yes, agreed. And this "cell" abstraction should also be used for implementing groups-in-groups / linked-groups.
